src/main/java/org/sagebionetworks/web/client/jsinterop/EntityFinderProps.java

```diff
@@ -52,7 +59,7 @@ public static EntityFinderProps create(
             List<EntityType> visibleTypesInList,
             List<EntityType> visibleTypesInTree,
             List<EntityType> selectableTypes,
-            String selectedCopy,
+            SelectedCopyHandler selectedCopy,
```

Comment:
This is the copy in the entity finder that says "Selected" or "Destination". We can now pass a `fn(int selectedCount)` to say "4 Items Selected"